File: app/src/main/java/mydebts/android/app/feature/main/MainRouter.java
package mydebts.android.app.feature.main;

import android.support.annotation.NonNull;

import mydebts.android.app.data.db.EventsTable;

public interface MainRouter {

    void navigateToEvents();

    void navigateToNewEvent();

    void navigateToEvent(@NonNull EventsTable event);

    void navigateBack();
}

File: app/src/main/java/mydebts/android/app/feature/main/MainRouter.kt
package mydebts.android.app.feature.main

import mydebts.android.app.data.model.Event
import mydebts.android.app.data.model.Person

interface MainRouter {

    fun navigateToEvents()

    fun navigateToNewEvent()

    fun navigateToEvent(event: Event)

    fun navigateBack()
}
